deps.edn

```diff
@@ -1,4 +1,4 @@
-{:paths ["src/main" "src/electron" "templates"]
+{:paths ["src/main" "src/electron" "templates" "src/resources"]
```

Comment:
I introduced a new directory for clj/cljs resources as templates is a misleading name for generic resources and unfortunately resources/ is already taken up for non clj/cljs resources. In a followup PR, I'd like to move the rest of templates/ to src/resources/templates/